proxy.go

```diff
@@ -121,13 +121,46 @@ func setupLogger(logLevel string) error {
 	return nil
 }
 
+func printAgentLogs(sock string) error {
+	// Don't return an error if nothing has been provided.
+	// This flag is optional.
+	if sock == "" {
+		return nil
+	}
+
+	agentLogsAddr, err := unixAddr(sock)
+	if err != nil {
+		proxyLog.Fatal("invalid agent logs socket address")
+		return err
+	}
+
+	conn, err := net.Dial("unix", agentLogsAddr)
+	if err != nil {
+		return err
+	}
+
+	go func() {
+		buf := make([]byte, 1024)
+		for {
+			if _, err := conn.Read(buf); err != nil {
```

Comment:
This would break agent logs with extra new lines when printing bellow. Please use bufio.ReadLine() or equivalent.

Reply:
Will do !